cpp/src/io/csv/csv_reader.cu

```diff
@@ -87,6 +87,8 @@ typedef struct raw_csv_ {
     bool* 				d_parseCol;		// device : array of booleans stating if column should be parsed in reading process: parseCol[x]=false means that the column x needs to be filtered out.
     long 				header_row;		// Row id of the header
     bool				dayfirst;
+	char				decimal;
+	char				thousands;
 } raw_csv_t;
```

Comment:
This file is obviously C++ (it uses std:: things), so it should define structs using C++ idiom: `struct Name { ... };`, NOT the C idiom `typedef struct Name_ { } Name;`